spock-specs/src/test/groovy/org/spockframework/smoke/VerifyMethodsSpecification.groovy

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.spockframework.smoke
+
+import org.spockframework.EmbeddedSpecification
+import org.spockframework.runtime.ConditionNotSatisfiedError
+
+class VerifyMethodsSpecification extends EmbeddedSpecification {
+
+  def setup() {
+    runner.throwFailure = false
+  }
+
+  def "@Verify method succeeds"() {
+    when:
+    def result = runner.runSpecBody '''
+@Verify
+private static void isPositive(int x) {
+    x > 0
+}
+
+def "feature"() {
+    expect:
+    isPositive(2)
+}
+'''
+
+    then:
+    result.failures.empty
+  }
+
+  def "@Verify method fails"() {
```

Comment:
You can use `@FailsWith(value = ConditionNotSatisfiedError, expectedMessage = "x > 0")` there is no need to use an embedded spec at all for the `@Verify` cases AFAICT.

Reply:
Done in [`5913512` (#2112)](https://github.com/spockframework/spock/pull/2112/commits/591351235e6817d5471f9a770ef726ebf187de62).

I did not do it initially because you need to put the entire exception message instead of just `x > 0`. This makes the annotation content longer than the entire feature method :D 

💭 If only it supported a pattern